cmd/root.go

```diff
@@ -44,6 +50,21 @@ var rootCmd = &cobra.Command{
 	// Uncomment the following line if your bare application
 	// has an action associated with it:
 	// Run: func(cmd *cobra.Command, args []string) { },
+	PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		if strings.HasPrefix(cmd.CommandPath(), "ybm completion") {
+			return
+		}
+		// Don't print any error if we are not able to fetch the latest release
+		latestVersion, err := releases.GetLatestRelease()
```

Comment:
This may break the json output .. Do we want to force people to update ?

Reply:
We are just encouraging the users to update to the latest version. Also, it doesn't break the json output.
<img width="1077" alt="Screen Shot 2023-03-22 at 9 13 24 PM" src="https://user-images.githubusercontent.com/48309285/226959118-8191a93b-063d-44c5-9123-1eb834e70db4.png">

Reply:
Verified that jq works

```
bash-5.2$ ./ybm cluster list -o pretty | jq .info
A newer version is available. Please upgrade to the latest version v0.0.17
{
  "cluster_endpoints": [
    {
      "accessibility_type": "PUBLIC",
      "host": "us-west3.9523227f-4290-4fe5-86cf-2dd8211f8b6f.gcp.devcloud.yugabyte.com",
      "id": "19b4036b-e4ac-46d9-9f7a-a8e38a582660",
      "pse_id": null,
      "region": "us-west3",
      "state": "ACTIVE",
      "state_v1": "ENABLED"
    }
  ],
  "cluster_network_version": "V0",
  "cluster_region_info_details": [
    {
      "id": "212789f0-aaf8-40d5-8bbb-162c9df0773b",
      "region": "us-west3"
    }
  ],
  "endpoints": {
    "us-west3": "us-west3.9523227f-4290-4fe5-86cf-2dd8211f8b6f.gcp.devcloud.yugabyte.com"
  },
  "health_info": {
    "notes": [],
    "state": "UNKNOWN"
  },
  "id": "9523227f-4290-4fe5-86cf-2dd8211f8b6f",
  "metadata": {
    "created_on": "2023-03-14T06:03:35.656653Z",
    "updated_on": "2023-03-22T07:36:07.350672Z"
  },
  "project_id": "f2a38352-2deb-4bb1-8948-0a2eed0dc70b",
  "read_replica_info": [],
  "software_version": "2.14.7.0-b51",
  "state": "PAUSED"
}
```

Reply:
Srinivas and I looked - the `logrus` logs get automatically sent to STDERR, this is why they don't interfere with jq.